owasp-zap/README.md

```diff
@@ -0,0 +1,39 @@
+# OWASP Zed Attack Proxy
+
+[ZAP](https://www.zaproxy.org/) is a security pen testing tool, which this project uses for active scanning for
+vulnerabilities. To manually run a scan similar to the ones integrated into [our GitHub Actions](../.github/workflows),
+run `./manual-scan.sh` from a command line. For full details on running the scan locally, see [Manual run ](#manual-run). Below is a brief explanation of the pieces of this integration and what they are for.
+
+# Authenticator
+
+The [authenticator](./authenticator) directory is a simple NodeJS script that leverages
+[aws-amplify](https://www.npmjs.com/package/aws-amplify) to quickly log a test user in and acquire an token for
+authentication. This script requires that the Cognit UserPool Client be configured to enable the SRP authentication
+flow, which can only be activated in pre-production environments.
```

Comment:
Can you explain a bit more here on why we can only activate it in pre-prod? Is this related to MFA requirements?